locust_plugins/jmeter_listener.py

```diff
@@ -28,6 +29,7 @@ def __init__(
         row_delimiter="\n",
         timestamp_format="%Y-%m-%d %H:%M:%S",
         flush_size=100,
+        results_filename=""
```

Comment:
probably makes more sense to default to None rather than empty string (and change the check to if results_filename: instead of if results_filename == "":